client/src/components/QA/Questions/Answers/AnswerList.jsx

```diff
@@ -5,14 +5,40 @@ import AListCSS from './AnswerList.module.css';
 class AnswerList extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      results: [
```

Comment:
`answers` would be more descriptive than `results`

Reply:
I wanted the data to match precisely for now, makes it easier to send data later if needed. But will change later if it becomes redundant. Thanks for feedback! Most of these things will be changed later. But would you merge when you get a min?